monasca_persister/README.rst

```diff
@@ -46,45 +46,63 @@ Those data points are mapped to NGSI Entities according to the following:
      - Monasca measurement
      - Monasca value_meta
      - NGSI attributes
-   * - region.allocated_ip
+   * - region.info
      - | name
        | location
        | latitude
        | longitude
        | ram_allocation_ratio
        | cpu_allocation_ratio
-     - region.allocated_ip
+       | {component}_version
```

Comment:
food for your mind, the number of OpenStack services that are deployed in one region is a no fixed number and can be increased or decreased in time. In fact take the example of Spain2 node.